src/textalloc/__init__.py

```diff
@@ -75,6 +77,10 @@ def allocate(
         avoid_label_lines_overlap (bool, optional): If True, avoids overlap with lines drawn between text labels and locations. Defaults to False.
         plot_kwargs (dict, optional): kwargs for the plt.plot of the lines if draw_lines is True.
         **kwargs (): kwargs for the plt.text() call.
+
+    Returns:
+        result_text_xy (List[Optional[Tuple[float, float]]]): List of optimal (x,y) arguments to use when calling plt.text to draw the label.
+        result_line (List[Union[Optional[Tuple[float, float], Tuple[float, float]]]]): List of optimal (x,y) arguments to use when calling plt.plot to draw the line.
```

Comment:
Change to:
    Returns:
        result_text_xy (List[Optional[Tuple[float, float]]]): List of resulting (x,y) positions for text labels used in the plt.text call.
        result_lines (List[Union[Optional[Tuple[float, float], Tuple[float, float]]]]): List of resulting (x,y) pairs used in the plt.plot call for drawing lines.